src/pom.xml

```diff
@@ -193,6 +197,12 @@
         <!-- Apache HTTP Client -->
         <apache-httpclient.version>4.5.3</apache-httpclient.version>
 
+        <!-- GeoTools -->
+        <geotools.version>17.3</geotools.version>
```

Comment:
Does this version depend on the used version of GeoServer resp. the used version of the importer?

Reply:
No, actually not.